docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -5,15 +5,18 @@
 <titleabbrev>Dense vector</titleabbrev>
 ++++
 
-The `dense_vector` field type stores dense vectors of float values. Dense
+The `dense_vector` field type stores dense vectors of values defined
+by `element_type` with `float` as a default. Dense
 vector fields can be used in the following ways:
 
 * In <<query-dsl-script-score-query,`script_score`>> queries, to score
 documents matching a filter
 * In the <<knn-search-api, kNN search API>>, to find the _k_ most similar
 vectors to a query vector
 
-The `dense_vector` type does not support aggregations or sorting.
+The `dense_vector` type does not support aggregations or sorting. When
+`element_type` is `byte` <<query-dsl-script-score-query,`script_score`>>
```

Comment:
Yes! I assumed that would be the next thing I work on once this work is done. I just didn't want to leave a lapse in the documentation in case for some reason something else gets higher priority.